src/sql/src/plan.rs

```diff
@@ -2042,15 +2042,17 @@ impl TryFrom<ClusterAlterOptionExtracted> for AlterClusterPlanStrategy {
 #[derive(Debug, Clone)]
 pub struct Params {
     pub datums: Row,
-    pub types: Vec<ScalarType>,
+    pub actual_types: Vec<ScalarType>,
```

Comment:
Not blocking, but what do you think about renaming this to `provided_types`? As far as I understand "actual_types" here refer to the types of the datums we were provided? The naming makes sense but I didn't realize the exact context until I saw the changes in `set_portal(...)`

Reply:
`provided_types` would be misunderstandable to mean that the types themselves were directly provided, i.e., a reader might think they refer to type hints (e.g., the `int` in `PREPARE b (int) AS SELECT $1`, which we don't support yet).

How about naming it `execute_types` + adding a comment on top of the field?

Reply:
`execute_types` makes sense to me! Thanks for thinking through this :)